runtime/src/apis.rs

```diff
@@ -237,6 +239,36 @@ impl_runtime_apis! {
 	}
 
 
+	impl frame_system_rpc_runtime_api::SystemEventsApi<Block> for Runtime {
+		fn fetch_transaction_success_status() -> Vec<frame_system_rpc_runtime_api::TransactionSuccessStatus> {
+			use frame_system_rpc_runtime_api::TransactionSuccessStatus;
+			use frame_system::Event;
+
+			let mut results: Vec<TransactionSuccessStatus> = Vec::new();
+			let event_records = System::read_events_no_consensus();
```

Comment:
Have you tested how much memory & time it takes for large block with large number of events? eg: with remark_with_events calls

Reply:
As discussed in the call, the feature is by default disabled and once it deployed to our testnet we will run some benchmarks to see how "bad" it can get.